### DISCUSSION TITLE
influxDBThresholdError More details required

### DISCUSSION THREAD

**Discussion by kamalshingala** [mderevyankoaqa/jmeter-influxdb2-listener-plugin#61] [Q&A] [answered]
Hi Michael, Thank you for building such a useful plugin. I was implementing this for one of my current performance test and wanting to understand a little bit more about influxDBThresholdError. From documentation, I can see that it’s configurable parameter and is not Malloy set to 5.

Can you please explain what do you mean by the error one by one reach 5 then it will stop importing data to InfluxDB. Do you mean the errors while sending data to InfluxDB or some other error?
Can you please explain a scenario when this can happen?
Thanks,
Kamal

**Comment by mderevyankoaqa** [comment, marked as answer]
Just to track history the answer is placed here https://github.com/mderevyankoaqa/jmeter-influxdb2-listener-plugin/issues/60

BR,
Mike
